index.js

```diff
@@ -23,66 +28,103 @@
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red")
+
+let thirstLevel = 12;
+
+function drinkWater(thirstLevel) {
+  console.log('Man I sure am thirsty');
+  thirstLevel--
+  console.log('Ahh that hits the spot');
+  return thirstLevel
+}
+
 
-// console.log(drinkWater(12))
+console.log(drinkWater(12))
 // // => "Man I sure am thirsty"
 // // => "Ahh that hits the spot"
 // // => 11
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red")
 
 
 
 // ***** Question 4 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+console.log("%cQuestion 4", "color: red")
+
+function sameSameButDifferent(num, maybeNum) {
+  if (num === maybeNum) {
+    return 'same same'
+  } else if (num === parseInt(maybeNum)) {
+    return 'same same (but different)';
+  } else {
+    return 'different';
+  }
+}
```

Comment:
Good. 
- If statement syntax ✅
- return statements ✅
- nice use of parseInt - You could have also used the == there.  example num == maybeNum ✅